README.md

```diff
@@ -1,15 +1,89 @@
 # Jubilant, the joyful library for integration-testing Juju charms
 
+![CI Status](https://github.com/canonical/jubilant/actions/workflows/ci.yaml/badge.svg)
+
 Jubilant is a Python library that wraps the [Juju](https://juju.is/) CLI for use in charm integration tests. It provides methods that map 1:1 to Juju CLI commands, but with a type-annotated, Pythonic interface.
 
 It was written to supersede the use of [pytest-operator](https://github.com/charmed-kubernetes/pytest-operator) and [python-libjuju](https://github.com/juju/python-libjuju/) for charm integration tests. Python-libjuju in particular has a complex and confusing API, and its use of `async` is unnecessary for testing.
 
-**NOTE:** Jubilant is in very early stages of development. This is pre-alpha code. Our intention is to release a 1.0.0 version early to mid 2025.
+Jubilant is currently in pre-release or "beta" phase ([PyPI releases](https://pypi.org/project/jubilant/#history)). Our intention is to release a 1.0.0 final version in May 2025.
+
+[**Read the full documentation.**](https://canonical-jubilant.readthedocs-hosted.com/)
+
+
+## Using Jubilant
+
+Jubilant is published to PyPI, so you can install and use it with your favourite Python package manager:
+
+```
+$ pip install jubilant
+# or
+$ uv add jubilant
+```
+
+To use Jubilant in Python code, instantiate a `Juju` instance and call methods like `deploy` and `wait`:
+
+```python
+import jubilant
+
+juju = jubilant.Juju()
+juju.deploy('snappass-test')
+juju.wait(jubilant.all_active)
+```
+
+Below is an example integration test [from Jubilant itself](https://github.com/canonical/jubilant/blob/main/tests/integration/test_basic.py). This test uses the [`juju` pytest fixture](https://github.com/canonical/jubilant/blob/main/tests/integration/conftest.py), which adds a model during setup, runs your test with a `Juju` instance pointing at that model, and destroys the model during teardown:
+
+```python
+def test_deploy(juju: jubilant.Juju):
+    juju.deploy('snappass-test')             # Deploy the charm
+    status = juju.wait(jubilant.all_active)  # Wait till the app and unit are 'active'
+
+    # Hit the Snappass HTTP endpoint to ensure it's up and running.
+    address = status.apps['snappass-test'].units['snappass-test/0'].address
+    response = requests.get(f'http://{address}:5000/', timeout=10)
+    response.raise_for_status()
+    assert 'snappass' in response.text.lower()
+```
+
+You don't have to use [Pytest](https://docs.pytest.org/en/stable/) with Jubilant, but it's what we usually recommend. Pytest's fixtures and its simple `assert`-based approach make writing tests easy.
```

Comment:
I feel like you could drop the "usually" here. It's probably more accurate, but I think for the README it invites unnecessary thought about what the unusual cases might be.

Reply:
Good call, done.